### PR TITLE
Output result not correct

### PR DESCRIPTION
**Pull request by BasantPandey**
Identity(42)
  .valueOf()
//=> 42 (Must be 42) not 33

Remove following lines

Identity(20)
  .concat(Identity(22))
  .valueOf()
//=> 35

due to this error message. Throwing error "TypeError: Identity.concat: Both containers must contain Semigroups of the same type"

### CODE REVIEW

File: docs/src/pages/docs/crocks/Identity.md [hunk 1]
```diff
@@ -333,12 +333,7 @@ import Identity from 'crocks/Identity'
 
 Identity(42)
   .valueOf()
-//=> 33
-
-Identity(20)
-  .concat(Identity(22))
-  .valueOf()
```
Comment (dalefrancis88) [inline docs/src/pages/docs/crocks/Identity.md:340]: I can see that this won't work, lets add in an example that does work. Each `Identity` will need to contain a SemiGroup of the same type

Comment (dalefrancis88) [reply docs/src/pages/docs/crocks/Identity.md:340]: Yeah both of those work great, i'll leave it to you to pick your fav, only request would be to copy similiar formatting to the one that didn't work

Comment (BasantPandey) [reply docs/src/pages/docs/crocks/Identity.md:340]: Sure I will Thanks

Comment (BasantPandey) [reply docs/src/pages/docs/crocks/Identity.md]: @dalefrancis88 Please approve my PR #472

File: docs/src/pages/docs/crocks/Identity.md [hunk 2]
```diff
@@ -333,12 +333,12 @@ import Identity from 'crocks/Identity'
 
 Identity(42)
   .valueOf()
-//=> 33
+//=> 42
 
-Identity(20)
-  .concat(Identity(22))
+Identity([10,20])
```
Comment (evilsoft) [inline docs/src/pages/docs/crocks/Identity.md:338]: Wondering why the linter is not picking this up, but gonna wanna add some spaces around the array values like:
`Identity([ 10, 20 ])`

**Comment by evilsoft** [comment]
@dalefrancis88 just going to merge this. Looks like all of your comments were addressed.

**Comment by dalefrancis88** [comment]
Easy done :)

**Pull request merged**